Formula/topgrade.rb

```diff
@@ -32,7 +32,7 @@ def install
 
     assert_match version.to_s, shell_output("#{bin}/topgrade --version")
 
-    output = shell_output("#{bin}/topgrade -n")
+    output = shell_output("#{bin}/topgrade -n --only brew")
```

Comment:
By default `topgrade` checks also `macOS system update`, and this could take time. So let's try to check only for brew upgrades and see if it help